fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgres/PostgresResourceNoProcDAO.java

```diff
@@ -194,7 +197,7 @@ public Resource insert(Resource resource, List<ExtractedParameterValue> paramete
      * @throws Exception
      */
     public long storeResource(String tablePrefix, List<ExtractedParameterValue> parameters, String p_logical_id, InputStream p_payload, Timestamp p_last_updated, boolean p_is_deleted,
-        String p_source_key, Integer p_version, Connection conn, ParameterDAO parameterDao) throws Exception {
+        String p_source_key, Integer p_version, String parameterHashB64, Connection conn, ParameterDAO parameterDao) throws Exception {
```

Comment:
Add @param tags for parameterHashB64, conn, parameterDao

Reply:
fixed